src/module_cache/WasmModuleCache.cpp

```diff
@@ -24,75 +24,26 @@ int WasmModuleCache::getCachedModuleCount(const std::string& key)
     return count;
 }
 
-std::string WasmModuleCache::getBaseCachedModuleKey(const faabric::Message& msg)
-{
-    std::string key = msg.user() + "/" + msg.function();
-    return key;
-}
-
-/**
- * There are two kinds of cached module here, the "base" cached module, i.e. the
- * default module with its zygote function executed, (same for all instances),
- * or one of many "special" cached modules, those restored from snapshots
- * captured at arbitrary points (e.g. when spawning a thread).
- */
```

Comment:
The `WasmModuleCache` was aware of snapshots, which added needless complexity and misdirection. Instead it now just holds a single cached module for each function, and knows nothing about any extra snapshotting that goes on _after_ that module has been used to clone others.